viewSQLposts.php

```diff
@@ -0,0 +1,37 @@
+<?php
+include ('include/SQLProject.php');
+?>
+
+
+<html>
+	<head>
+		<title> View Blog Posts </title>
+		<link rel='stylesheet' href='maindesign.css' />
+	</head>
+</html>
+
+<?php
+include('config/config.php');
+include('include/db_query.php');
+
+
+function GetAllBlogposts($blogpostId){
+	$result = dbQuery("
+		SELECT *
+		FROM Blogposts
+		WHERE BlogpostId = $blogpostId
+		")->fetch();
+	return $result;
+}
+
+
+$test=GetAllBlogposts($_REQUEST['postId']);
```

Comment:
I'd update the name of this function. It looks like originally it was getting all blog posts, but now it's just getting one.